src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -3,24 +3,37 @@
 #include "controlobject.h"
 #include "util/math.h"
 
-#define kConfigKey "[ReplayGain]"
-
-static const int kReplayGainReferenceLUFS = -18;
-
-
-DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
+namespace {
+const char* kConfigKey = "[ReplayGain]";
```

Comment:
Since your first comment, I have introduced the ReplayGainSettings class to manage the settings.
So the main issue is solved. 
These remaining strings here are required to setup the Control Objects. Everywhere else in Mixxx we use new strings to reference them, so I like to keep them here.